src/Akeneo/Bundle/ElasticSearchBundle/phpspec.yml

```diff
@@ -0,0 +1,5 @@
+suites:
```

Comment:
to launch the specs as a standalone package
(not really needed right, but could be)